civihr_employee_portal/js/reports.js

```diff
@@ -202,16 +202,6 @@
     fromElement.detach().appendTo(toElement);
   };
 
-  /**
-   * Make the report field rows container at least the same height as the field
-   * selection.
-   */
-  HRReport.prototype.makeReportFieldRowsContainerFullHeight = function () {
-    var fullHeight = this.pivotTableContainer.find('.report-fields-selection table').height();
-
-    this.pivotTableContainer.find('.report-field-rows table').css('min-height', fullHeight);
-  };
-
```

Comment:
Why is this  function removed? Don't we need it any more?

Reply:
This was added and removed by me. At first the field rows' height was done dynamically, but after fixing the fields list I removed the JS code and added a CSS fixed max-height.